packages/components/src/components/hds/badge/index.ts

```diff
@@ -4,39 +4,44 @@
  */
 
 import Component from '@glimmer/component';
-import { assert } from '@ember/debug';
-
-export const DEFAULT_SIZE = 'medium';
-export const DEFAULT_TYPE = 'filled';
-export const DEFAULT_COLOR = 'neutral';
-export const SIZES = ['small', 'medium', 'large'];
-export const TYPES = ['filled', 'inverted', 'outlined'];
-export const COLORS = [
-  'neutral',
-  'neutral-dark-mode',
-  'highlight',
-  'success',
-  'warning',
-  'critical',
-];
-
-export default class HdsBadgeIndexComponent extends Component {
+import {assert} from '@ember/debug';
+import {type Color, ColorValues, type Size, SizeValues, type Type, TypeValues} from "./types.ts";
```

Comment:
In other PRs (eg. [here](https://github.com/hashicorp/design-system/pull/1984/files#diff-b321dcb3b8e11ce30d48a349345e73fe4a3b35eef62e2ea1d20547cf3dc5ecec)) I see this formatted as below. 

Plus, in the text the types are prefixed with `Hds[ComponentName]` while here they're not.

@alex-ju do you think it would be possible to settle on one common formatting (including order) and naming convention?  /cc @hashicorp/hds-engineering

```suggestion
import {
  ColorValues,
  SizeValues,
  TypeValues,
  type Color,
  type Size,
  type Type,
} from "./types.ts";
```